content/blog/2024/202312-review.rst

```diff
@@ -0,0 +1,37 @@
+========================================
+202312 review
```

Comment:
@kuomc , please provide a better title

Reply:
`sciwork December 2023 review: weekly meetup and Taipei scisprint`

Reply:
Quite precise!